src/axom/sina/config.hpp.in

```diff
@@ -0,0 +1,13 @@
+#ifndef SINA_CONFIG_HPP_
+#define SINA_CONFIG_HPP_
+
+/// \file
+
+#define SINA_VERSION_MAJOR @SINA_VERSION_MAJOR@
+#define SINA_VERSION_MINOR @SINA_VERSION_MINOR@
+#define SINA_VERSION_PATCH @SINA_VERSION_PATCH@
+
+#define SINA_VERSION "@SINA_VERSION_MAJOR@.@SINA_VERSION_MINOR@.@SINA_VERSION_PATCH@"
```

Comment:
This was one of the last things I added in as I wasn't entirely sure where we were going with this. Should we just remove this entirely and give a completely fresh start on versioning Sina with Axom? Does Axom have plans to version its individual components?

Reply:
I'm asking this as I'm not sure if it's possible to mark defines as deprecated. I'll see if I can find something like that to work off of. If that's the case then we can just go that route instead

Reply:
If users aren't already a `SINA_VERSION` define, I'd recommend removing it entirely and letting users check the axom version.

We decided not to have different versions for axom components since we don't want to support components from different versions inter-operating (which would be a huge headache w/o any clear advantage).

Reply:
Great point on the inter-operation. I'd need to double check, but I think SINA_VERSION on the C++ side was only used for spack/making tarballs vs. any user scooping it up.

One note: the underlying file format does have a version, and if the format has any backwards-incompatible changes, it'll be important for the C++ component to include it for other tools to pick up. Through now the format and Sina versions have been treated as one in the same just due to how the releases work. So you should be good to replace this with the Axom version for all build/version-related reasons, but might want to keep this number someplace and stick the word `_FORMAT_` in it, to be updated solely for changes in file writing logic.

Reply:
Maybe I should change the above versioning to:
```
#define SINA_FILE_FORMAT_VERSION_MAJOR @SINA_FILE_FORMAT_VERSION_MAJOR@
#define SINA_FILE_FORMAT_VERSION_MINOR @SINA_FILE_FORMAT_VERSION_MINOR@
#define SINA_FILE_FORMAT_VERSION_PATCH @SINA_FILE_FORMAT_VERSION_PATCH@

#define SINA_FILE_FORMAT_VERSION "@SINA_FILE_FORMAT_VERSION_MAJOR@.@SINA_FILE_FORMAT_VERSION_MINOR@.@SINA_FILE_FORMAT_VERSION_PATCH@"
```
instead then?